src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        int supply = 0;
+        int buy = 0;
+        int result = 0;
+        try (BufferedReader reader = new BufferedReader(new FileReader(new File(fromFileName)));
+                BufferedWriter writer = new BufferedWriter(new FileWriter(new File(toFileName)))) {
+            String line = reader.readLine();
+            while (line != null) {
+                System.out.println(line);
+                String[] separatedLine = line.split(",");
+                if (separatedLine[0].equals("supply")) {
+                    supply += Integer.parseInt(separatedLine[1]);
+                } else if (separatedLine[0].equals("buy")) {
+                    buy += Integer.parseInt(separatedLine[1]);
+                }
+                line = reader.readLine();
+            }
+            result = supply - buy;
+            writer.write("supply," + supply + "\n");
```

Comment:
[Use StringBuilder instead of String to concat data in loops](https://github.com/mate-academy/jv-work-with-file/blob/master/checklist.md#use-stringbuilder-instead-of-string-to-concat-data-in-loops)

Reply:
but where is an usage of String to concat in a loop here?